src/chrome/content/rules/General-anzeiger-bonn.de.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="General-anzeiger-bonn.de">
+	<target host="general-anzeiger-bonn.de" />
+	<target host="www.general-anzeiger-bonn.de" />
```

Comment:
Please also add `trauer.general-anzeiger-bonn.de` and `epaper.general-anzeiger-bonn.de`.